mpyg321/mpyg321.py

```diff
@@ -227,6 +227,11 @@ def silence(self):
         """Silences the player"""
         self.player.sendline("SILENCE")
 
+    def load_list(self, entry, url):
+        """Load an entry in a list"""
+        self.player.sendline("LOADLIST {} {}".format(entry, url))
```

Comment:
Can you please provide me with a valid URL so that I can test this feature?

Reply:
LOADLIST seem specific to mpg123 

```
grep strcasecmp remote.c (mpg321 source code)
    if (strcasecmp(input, "L") == 0 || strcasecmp(input, "LOAD") == 0)
    else if (strcasecmp(input, "J") == 0 || strcasecmp(input, "JUMP") == 0)
    else if (strcasecmp(input, "S") == 0 || strcasecmp(input, "STOP") == 0)
    else if (strcasecmp(input, "Q") == 0 || strcasecmp(input, "QUIT") == 0)
    else if (strcasecmp(input, "P") == 0 || strcasecmp(input, "PAUSE") == 0)
    else if (strcasecmp(input, "G") == 0 || strcasecmp(input, "GAIN") == 0)

```